client/core/core_test.go

```diff
@@ -2680,7 +2714,7 @@ func TestTradeTracking(t *testing.T) {
 	auditInfo.expiration = matchTime.Add(tracker.lockTimeTaker)
 
 	// success, full handleAuditRoute>processAuditMsg>auditContract
-	rig.db.updateMatchChan = make(chan order.MatchStatus, 1)
+	rig.db.updateMatchChan = make(chan order.MatchStatus, 2)
```

Comment:
One of a few larger test buffers to capture the extra synchronous send without deadlock.